### ISSUE TITLE
Assessing Game Balance with AlphaZero: Exploring Alternative Rule Sets in Chess

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#106]
# TL;DR

This is an attempt to assess game balance using AlphaZero. To train AlphaZero on the chess mimics with minor rule changes such as no castling, we can see how experienced players view the game. In the history of chess, the rules have been repeatedly modified and settled on the current rules. Other games can be made to simulate such a thing with AlphaZero, they said.

<img width="695" alt="(e) An example from Torpedo chess White needs to generate" src="https://user-images.githubusercontent.com/24524018/93066534-129cc980-f6b5-11ea-8c59-ad094437ea43.png">

# Why it matters:

# Paper URL
https://arxiv.org/abs/2009.04374

# Submission Dates(yyyy/mm/dd)

# Authors and institutions
Nenad Tomašev, Ulrich Paquet, Demis Hassabis, Vladimir Kramnik
- DeepMind

# Methods

# Results

# Comments
